internal/status/http-statuses.csv

```diff
@@ -40,6 +40,9 @@
 416,4,Requested Range Not Satisfiable,RFC7231
 417,4,Expectation Failed,RFC7231
 418,4,I'm a teapot,RFC2324
+419,4,Page Expired,Unofficial
+420,4,Enhance your calm,Unofficial
+420,4,Method Failure,Unofficial
```

Comment:
I made this PR https://github.com/sterchelen/hssp/pull/12 to handle multiple code definitions for a single code.